bootstrap/sql/mysql/v004__create_db_connection_info.sql

```diff
@@ -56,3 +56,8 @@ ALTER TABLE role_entity
 ADD COLUMN `default` BOOLEAN GENERATED ALWAYS AS (JSON_EXTRACT(json, '$.default')),
 ADD INDEX(`default`);
 
+-- Add tag label source
+ALTER TABLE tag_usage
+ADD COLUMN source TINYINT NOT NULL FIRST, -- Source of tag (either from TagCategory or Glossary)
```

Comment:
@sureshms we need add a default here for the upgrades to work. I think adding TagCategory as default is safe option

Reply:
Default value in the schema for the new field `source` is `tag` which is what was the previous value. I think it should get set to `tag` for all the existing installations. If not, the COLUMN would be null and the system would not startup.

Reply:
Actually you may be right about setting the default field value. The default value is only in the JSON schema definition which the table JSON column does not have. Let me fix it.